azerothcore/azerothcore-wotlk
# Fix (DB/Scripts)  npc speaking in IcecrownCitadel links to BroadcastText

## Changes Proposed:
-  
-  

## Issues Addressed:

- Closes 

## SOURCE:


## Tests Performed:




## How to Test the Changes:

1. Use a client other than English
1. quest add 24547
2. Enter IcecrownCitadel
3. Wait for the Lich King to whisper randomly
4. Observe whether the Lich King whisper displays text in the language of the corresponding client.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
